src/firmware/app/control/trajectory_planner_impl_test.cpp

```diff
@@ -51,15 +51,15 @@ TEST(TrajectoryPlannerImplTest, test_robot_state_generation_at_constant_t_interv
         segment_lengths_angular);
 
     const float delta_t = (path_parameters.t_end - path_parameters.t_start) /
-                          (path_parameters.num_elements - 1);
+                          (float)(path_parameters.num_elements - 1);
```

Comment:
nit: we prefer `static_cast` in all `.cpp` files

Reply:
fixed